### PR TITLE
Drop support for unmaintained Julia 0.5

### PR DESCRIPTION
**Pull request by giordano**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -65,9 +65,9 @@ end
 
     # test relax keyword
     faulty_header = replace(header, "RADESYS = 'ICRS'", "RADECSYS= 'FK5' ")
-    wcs = WCS.from_header(faulty_header)[1]  # default is maximum relaxation
+    global wcs = WCS.from_header(faulty_header)[1]  # default is maximum relaxation
```
Comment (kbarbary) [inline test/runtests.jl]: Why the `global` here?

Comment (giordano) [reply test/runtests.jl]: A warning emitted by julia 0.7 suggests putting the `global` when setting a global variable

Comment (kbarbary) [reply test/runtests.jl]: Huh, seems like a bad warning. Can you instead rename these `wcs` variables to something else to get rid of the warning? (e.g., `wcs_relaxed`, and `wcs_strict` two lines below) We don't actually need to assign to the `wcs` binding earlier in the file, so `global` is a bit confusing here.

Comment (giordano) [reply test/runtests.jl]: Done!

**Comment by kbarbary** [comment]
I'm not sure what's going on with the OS X workers on Travis getting an InterruptException while building wcslib.

**Pull request merged**